content/en/tracing/service_catalog/guides/integrating-with-ci-visibility.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Integrating with CI Visibility
+kind: guide
+further_reading:
```

Comment:
can we add another link into this further_reading table?

link: "/continuous_integration/search/?tab=pipelines"
tag: "Documentation"
text: "Datadog CI Pipeline Visibility